<launch>
    <include file="$(find applevision_rospkg)/launch/frames.launch" />
    <node name="tf2_proxy_run" pkg="applevision_rospkg" type="tf2_proxy_run.py" respawn="true" />
    <node name="cam_image_proc" pkg="image_proc" type="image_proc" ns="palm_camera" />
    <group ns="palm_camera">
        <node name="camera_info" pkg="rostopic" type="rostopic"
            args="pub camera_info sensor_msgs/CameraInfo
            '{header: {seq: 0, stamp: {secs: 0, nsecs: 0}, frame_id: 'palm'},
            height: 360, width: 640, distortion_model: 'plumb_bob',
            D: [0.06133711, -0.00090462, 0.002165, -0.00302198, -0.30672296],
            K: [552.09305325, 0.0, 333.68780409, 0.0, 555.91052977, 162.88250186, 0, 0, 1],
            R: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0],
            P: [552.09305325, 0.0, 333.68780409, 0.0, 0.0, 555.91052977, 162.88250186, 0, 0.0, 0.0, 1.0, 0.0],
            binning_x: 0, binning_y: 0,
            roi: {x_offset: 0, y_offset: 0, height: 360, width: 640, do_rectify: false}}' -r 2"
            output="screen"/>
    </group>
</launch>